packages/landingpage-new/pages/getting-started/index.tsx

```diff
@@ -1,19 +1,19 @@
 import { useRouter } from 'next/router';
 import { useMount } from 'react-use';
-import ReactGuidePage from './react';
+import JsGuidePage from './javascript';
 import { Framework } from '../../utils/frameworks';
 
 /**
- * Forwards to react guide as default
+ * Forwards to javascript guide as default
  */
 const GettingStarted = () => {
   const { pathname, replace } = useRouter();
 
   useMount(() => {
-    replace(pathname + '/' + Framework.REACT);
+    replace(pathname + '/' + Framework.JS);
```

Comment:
The react guide will be probably be the most popular one. That's why I think we should keep routing to the react one by default (but no need to change the order tho).